### DISCUSSION TITLE
Settings for 400 instance images?

### DISCUSSION THREAD

**Discussion by kaylazy** [TheLastBen/fast-stable-diffusion#1281] [Q&A]
I'm training a model to a produce a specific type/style of photography collages. The photos do contain people as subjects but I don't care about training on the subjects, just the style of images. It's kinda like a before and after photo thing.

I want to use 400 images to add more variation to the model, I don't want to settle for any less.

I was wondering what unet and text encoder learning rate to use and how many steps for each to use when training on 400 instance images?

**Comment by TheLastBen** [comment]
Set the text encoder to 350 and the unet to 1200, then test the model, use a good prompt.

**Reply by kaylazy** [reply]
I was originally thinking to do encoder 300 and then 5000 steps unet. Then I was going to add 20,000 more steps with the style box ticked. I assumed this was the way to go about it. I'm glad I read this before I waited for those 20,000 steps to finish! Lol.

So do I only use 1200 unet steps since I am focusing on a style and not a specific person? or what is the reason you think this is the best option?

**Reply by TheLastBen** [reply]
the learning rate of 1e-5 requires very few steps, you will get even better results with the V2-512

**Reply by kaylazy** [reply]
my results arent as great as they were with smaller datasets. It's not even hitting close to the style, whereas before they were always close enough. I suspect I might need 3000 steps or so since thats what I used on the smaller sets that had better results. Should I tick Style_Training if I am going to train the model further?

**Reply by TheLastBen** [reply]
My recommendation is to not go above 20 images for a style, but since you said  "I don't want to settle for any less", then you're gonna have to try various things to get it to work

**Reply by gilroff** [reply]
selecting the best 20 and adding the rest to concept images and using a high number of concept image training steps. will this work?

**Reply by TheLastBen** [reply]
yes, is should work
